tests/integration/test_integration.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Integration tests for the nms-magmalte rock."""
+
+import docker  # type: ignore[import]
+import yaml
+import requests  # type: ignore[import]
```

Comment:
You could avoid this ignore by adding `types-requests` to the `tox.ini`.

Reply:
✅